aws/rust-runtime/aws-config/src/lib.rs

```diff
@@ -466,12 +488,20 @@ mod loader {
                 ))
             };
 
+            let credentials_cache = if let Some(cache) = self.credentials_cache {
+                cache
+            } else {
+                let mut builder = CredentialsCache::lazy_builder().time_source(conf.time_source());
+                builder.set_sleep(conf.sleep());
+                builder.into_credentials_cache()
+            };
```

Comment:
IMO, using `.unwrap_or_else()` is preferred to having two assignments on one line.

Reply:
Agreed. Addressed in af61b54. Tried to convert other occurrences in the same method but they slightly differ and could not be converted, e.g. some of them have `.await` in their else blocks.